core/trino-main/src/main/java/io/trino/sql/planner/LocalExecutionPlanner.java

```diff
@@ -581,7 +581,8 @@ public LocalExecutionPlan plan(
         PartitionFunction partitionFunction;
```

Comment:
Is it somewhat testable in query plans or similar? Is it worth testing? I would assume there was regression due to scaling rebalancer in FTE

Reply:
I will check how much regression we get realistically - not tested - yet this code is not needed in FTE - so CPU cycles are wasted for sure.